ipyvolume/headless.py

```diff
@@ -59,16 +72,19 @@ def _screenshot_data(
     result = chrome.Runtime.evaluate(expression='ipvss()')
     tries = 0
     while tries < 10:
+        #print(str(result))
         try:
-            url = result['result']['result']['value']
+            url = result[0]['result']['result']['value']
```

Comment:
why did this change, and which version, and can we make it detect this? so it also works on older versions?

Reply:
Actually I don't know why/when this has changed. This seems to work fine but recently throwing the following error: "tuple indices must be integers or slices, not str". This is the main reason prompting this pull request.